### PR TITLE
homework 3 - conditions and loops

### PR DESCRIPTION
**Pull request by LesiaDorofiienko**

### CODE REVIEW

File: conditions-and-loops/conditions1.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const averageGrade = (Math.random() * 100).toFixed();
```
Comment (sttaran) [inline conditions-and-loops/conditions1.js]: averageGrade is type of string, convert it to number

File: conditions-and-loops/conditions1.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const message = `Average grade: ${averageGrade} - `
+
+if (averageGrade <= 60) {
```
Comment (sttaran) [inline conditions-and-loops/conditions1.js]: averageGrade < 60

File: conditions-and-loops/conditions2.js
Comment (sttaran) [inline conditions-and-loops/conditions2.js:1]: image

**Review by sttaran — CHANGES_REQUESTED** [review]

File: conditions-and-loops/conditions1.js (hunk 1 above)
Comment (LesiaDorofiienko) [reply conditions-and-loops/conditions1.js]: comparison operators transform `averageGrade` to string automatically, and in this example `averageGrade` is an integer, so it's completely safe
https://javascript.info/comparison#comparison-of-different-types

**Review by sttaran — APPROVED** [review]

**Pull request merged**